src/map/entities/charentity.cpp

```diff
@@ -3098,6 +3098,26 @@ bool CCharEntity::OnAttackError(CAttackState& state)
     return false;
 }
 
+bool CCharEntity::isInTriggerArea(uint32 triggerAreaID)
+{
+    return m_InsideTriggerAreaID & (1 << triggerAreaID);
```

Comment:
I just realized that it might make sense to change the name of this, since it's no longer a single ID.